_pages/events/2025-03-27_project-end.md

```diff
@@ -0,0 +1,62 @@
+---
+layout: single
+classes: wide
+title: HPC Ready - Developing RSE and HPC skills
+ 
+permalink: /events/2025-03-27_project-end/
+author_profile: false
+
+
+---
+
+**Join us for "HPC Ready: Developing RSE and HPC skills" on Thursday 27th March 2025 from 10:00 to 16:00 UTC**, which will take place **in person at the University of Edinburgh [Bayes Centre](https://bayes-centre.ed.ac.uk/), and online**.
```

Comment:
Just a thought, the time makes it obvious but we could perhaps highlight that it's a one-day workshop since I feel that tends to engage people somewhat? e.g. 
```suggestion
**Join us for a one-day workshop - "HPC Ready: Developing RSE and HPC skills" - on Thursday 27th March 2025 from 10:00 to 16:00 (GMT/UTC)**, which will take place **in person at the University of Edinburgh [Bayes Centre](https://bayes-centre.ed.ac.uk/), and online**.
```